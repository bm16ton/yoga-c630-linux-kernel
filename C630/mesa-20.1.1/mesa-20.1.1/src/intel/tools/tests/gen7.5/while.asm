while(8)        JIP: -20                                        { align1 1Q };
while(16)       JIP: -20                                        { align1 1H };
while(8)        JIP: -30                                        { align16 1Q };
(-f0.0) while(8) JIP: -48                                       { align1 1Q };
(-f0.0) while(16) JIP: -48                                      { align1 1H };
(-f0.0.x) while(8) JIP: -48                                     { align16 1Q };
