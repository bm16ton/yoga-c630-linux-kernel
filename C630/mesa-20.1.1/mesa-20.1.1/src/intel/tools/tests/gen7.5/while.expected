27 00 60 00 84 3c 00 20 00 00 8d 00 ec ff 00 00
27 00 80 00 84 3c 00 20 00 00 8d 00 ec ff 00 00
27 01 60 00 84 3c 0f 20 04 00 6e 00 e2 ff 00 00
27 00 71 00 84 3c 00 20 00 00 8d 00 d0 ff 00 00
27 00 91 00 84 3c 00 20 00 00 8d 00 d0 ff 00 00
27 01 72 00 84 3c 0f 20 04 00 6e 00 d0 ff 00 00
